.travis.yml

```diff
@@ -49,9 +49,13 @@ script:
 
 matrix:
   include:
-    - php: 7
-      env: "DB=pgsql PHP_COVERAGE=TRUE"
-    - php: 7
-      env: "DB=mysql"
+    - php: 7.1
+      env: DB=sqlite CORE_BRANCH=master
+    - php: 7.2
+      env: DB=sqlite CORE_BRANCH=master
+    - php: 7.1
```

Comment:
stable10 php7.1-sqlite and php7.2-sqlite are already in the test environments. These two matrix entries are redundant. See https://github.com/owncloud/twofactor_totp/blob/01edf7bb19717ec30c5b15951d88dd893e5ff400/.travis.yml#L7 . 
But there is no test for postgresql, mysql and oracle. IMHO, we should add these databases to the test matrix.